src/main/java/de/caritas/cob/consultingtypeservice/config/SecurityConfig.java

```diff
@@ -76,7 +76,7 @@ protected void configure(final HttpSecurity http) throws Exception {
         .requestMatchers(new AntPathRequestMatcher("/topic/*"))
         .authenticated()
         .requestMatchers(new AntPathRequestMatcher("/topic-groups"))
-        .authenticated()
+        .permitAll()
```

Comment:
it should have same permissions as topic IMO, permit all? are you not planning to add some management operations there?

Reply:
Actually, I can't tell much here, but I was told by @web-mi to do so. :shrug:

Reply:
Currently we will not integrate the topic groups inside the admin console.